ui/src/app/views/account-creation/component.html

```diff
@@ -14,11 +14,33 @@ <h2>Create your account</h2>
       </div>
     </div>
     <div class="form-section">
-      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Create Account">
-      <h5 style="display: inline-block;">@{{gsuiteDomain}}</h5>
+      <input type="text" class="short-input" id="username" name="username" [(ngModel)]="profile.username" (ngModelChange)="usernameChanged()" placeholder="Username">
+      <h5 style="display: inline-block;" class="domain">@{{gsuiteDomain}}</h5>
+      <clr-tooltip>
+        <clr-icon clrTooltipTrigger shape="info-standard" class="is-solid info-icon"></clr-icon>
+        <clr-tooltip-content clrPosition="top-right" clrSize="lg" *clrIfOpen>
+          <div>
+            Usernames can contain letters (a-z), numbers (0-9), dashes (-), underscores (_),
+            apostrophies ('), and periods (.).
+          </div>
+          <br>
+          <div>
+            Usernames can't contain an ampersand (&), equal sign (=), brackets (<,>), plus sign (+),
+            comma (,), or more than one period (.) in a row.
+          </div>
+          <br>
+          <div>
+            Usernames can begin or end with non-alphanumeric characters except (.), with a maximum
```

Comment:
Also in the Google documentation, so it's OK if we want to keep this, but I guess this is just saying that a username cannot start or end with a "."?